.travis.yml

```diff
@@ -6,9 +6,9 @@ before_install:
     - "sh -e /etc/init.d/xvfb start"
 install:
     - "pip install -e git://github.com/edx/xblock-sdk.git#egg=xblock-sdk"
-    - "pip install -r $VIRTUAL_ENV/src/xblock-sdk/requirements.txt"
-    - "pip install -r $VIRTUAL_ENV/src/xblock-sdk/test-requirements.txt"
+    - "make -C $VIRTUAL_ENV/src/xblock-sdk install"
     - "python setup.py develop"
-script: pep8 leaderboards --max-line-length=120 && pylint leaderboards && python run_tests.py --with-coverage --cover-package=leaderboards
+    - "mkdir var"  # for var/workbench.log
```

Comment:
@smarnach I had to create a `var` dir to prevent the logging config from complaining about ValueError: Unable to configure handler 'logfile': [Errno 2] No such file or directory: '.../var/workbench.log'.

This doesn't seem like the best place to do this.  Should there be a dummy var file in the repo?  Or can I configure the workbench.log file location to write to the root dir?